Libraries/CustomComponents/Navigator/Navigator.js

```diff
@@ -1087,6 +1101,7 @@ var Navigator = React.createClass({
     var presentedRoute = this.state.routeStack[this.state.presentedIndex];
     var popSceneConfig = this.props.configureScene(presentedRoute); // using the scene config of the currently presented view
     this._enableScene(popIndex);
+    this._clearTransformations(popIndex);
```

Comment:
Can you please add a comment why this is needed so people reading the code in the future understand why it's needed to reset the transform here?